go-controller/cmd/ovn-k8s-cni-overlay/app/helper_windows.go

```diff
@@ -109,7 +109,7 @@ func deleteHNSEndpoint(endpointName string) error {
 // Small note on this, the call to this function should be idempotent on Windows.
 // The fact that CNI add should be idempotent on Windows is stated here:
 // https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/cni/cni_windows.go#L38
-// TODO: add support for custom MTU
+// TODO: add proper MTU config (GetCurrentThreadId/SetCurrentThreadId) or via OVS properties
```

Comment:
What's the downside of this method? We are still in the host namespace anyway.

Reply:
W.r.t. time:
in-process method/dll binding < Spawning new processes on Windows is expensive < Powershell is rather slow compared to normal binaries, esp. when using WQL (filters).

I hope the above makes sense, if not I can expand a bit more on the explanation.

Reply:
I got it. Thanks.